### ISSUE TITLE
About training settings

### ISSUE THREAD

**Issue by kcs6568** [giddyyupp/coco-minitrain#12]
Hi. Thank you for the great project!

I'm curious about the training settings for each model epoch, learning rate, etc.

Can I get information about each model? (Faster RCNN, RetinaNet, etc)

**Comment by nerminsamet** [comment]
Hello, 

In order to train the Faster RCNN, MaskRCNN and RetinaNet models we used [MMDetection](https://github.com/open-mmlab/mmdetection) with the default parameter settings. Working with [Detectron2](https://github.com/facebookresearch/detectron2) also should be OK. For the training of ExtremeNet, CornerNet and HoughNet we used the official repositories again with the default parameter settings. For some models we have to work with smaller batch sizes. In such cases we followed the linear learning rate rule to scale the learning rate according to batch size.

**Comment by kcs6568** [comment]
Great! Thanks for the reply!
